### ISSUE TITLE
NIP checker issue

### ISSUE THREAD

**Issue by GodsHand86** [kolton/d2bot-with-kolbot#1504]
I have tried both 32bit and 64bit version. I have tried the GUP.exe, it says there's an update, but it then just downloads and installs the same version 7.6.6. I have NO APIs folder to play the NIP.xml file in.

**Comment by tlwh163** [comment]
must 32bit notepad++

Download must includes 3 files:
NipCheck.dll
NIP.xml
userDefineLanguage.xml

Navigate to:
C:\Program Files (x86)\Notepad++\plugins\   <-- NipCheck.dll

Navigate to: (If the folder does not exist, Create )
C:\Program Files (x86)\Notepad++\plugins\APIs\  <-- NIP.xml

Navigate to:
C:\Users\<your user>\AppData\Roaming\Notepad++\ <-- userDefineLang.xml 

so, If the folder does not exist, Create...

**Comment by GodsHand86** [comment]
Thank you for posting the directions, I had not read them multiple times trying to figure this out. I've also tried creating the folder, even with this done it does not show up as a plugin in Notepad++.

**Comment by mf022** [comment]
Did you see https://github.com/blizzhackers/pickits/tree/master/nipcheck

About red icons on files ... you are referring to tortoiseSVN colors for changed files?

**Comment by GodsHand86** [comment]
Yes I have seen those directions as well, I've pretty much given up on this subject. 

Yes I recently figured out that it just means I've edited the files containing a red !, thank you.

**Comment by Kgc-Jr** [comment]
How about a 64-bit version for Notepad++ x64...

**Comment by Teffeq** [comment]
Hi. I am having the same issue. I tried 32 bit. 64 bit. latest  versions.
Nothing works. There is no APIs folder.
The worst is, I tried creating a new folder, renaming it APIs but it didnt work.

So now everytime i get an error... i have to spend so much time just to know where the error is.....
